README.md

```diff
@@ -18,115 +19,24 @@ See live examples of GOV.UK Frontend components, and guidance on when to use
-
-```
-npm install nunjucks --save
-```
-
-### Installation
-
-GOV.UK Frontend is currently in private beta. You will need to log in to NPM
-using credentials provided by the Design System team.
-
-Run the following command to log in:
-```
-npm login
-```
 
-Enter the username, password and email address you were provided with, when
-prompted.
+## Quick start
 
-You can install all components or one or more individual components depending on
-your needs.
+There are 2 ways to start using GOV.UK Frontend.
 
-To install all components, run:
-
-```
-npm install --save @govuk-frontend/frontend
-```
-
-After you have installed GOV.UK Frontend the `@govuk-frontend` package will
-appear in your `node_modules` folder.
-
-### Import styles
-
-You need to import the GOV.UK Frontend styles into the main Sass file in your
-project. You should place the below code before your own Sass rules (or Sass
-imports) if you want to override GOV.UK Frontend with your own styles.
-
-1. To import all components, add the below to your Sass file:
-
-  ```CSS
-  @import "node_modules/@govuk-frontend/frontend/all";
-  ```
-
-2. To import an individual component (for example a button), add the below to
-your Sass file:
-
-  ```CSS
-  @import "node_modules/@govuk-frontend/frontend/components/button/button";
-  ```
-
-#### Optional: Resolve import paths
-
-If you wish to resolve the above `@import` paths in your build (in order to
-avoid prefixing paths with `node_modules`), you should add `node_modules` to
-your [Sass include paths](https://github.com/sass/node-sass#includepaths)
-(in Ruby, they should be added to [assets
-paths](http://guides.rubyonrails.org/asset_pipeline.html#search-paths)).
-
-For example, if your project uses Gulp, you would add the Sass include paths to
-your Gulp configuration file (for example `gulpfile.js`) with
-[gulp-sass](https://www.npmjs.com/package/gulp-sass). Below is an example:
-
-```JS
-gulp.task('sass', function () {
-  return gulp.src('./sass/**/*.scss')
-    .pipe(sass({
-      includePaths: 'node_modules'
-     }))
-    .pipe(gulp.dest('./css'));
-});
+Once installed, you will be able to use the code from the examples in the
+[GOV.UK Design System](https://govuk-design-system-production.cloudapps.digital/)
+in your service.
 
-```
+### 1. Install with npm (recommended)
 
-(If you compile Sass to CSS in your project, your build tasks will already
-include something similar to the above task - in that case, you will just need
-to include add `includePaths` to it.)
+We recommend [installing GOV.UK Frontend using node package manager
+(npm)](docs/installation/installing-with-npm.md).
 
-After resolving the import paths you can import GOV.UK Frontend by using:
+### 2. Install by using built files
 
-```CSS
-@import "@govuk-frontend/frontend/components/button/button";
-```
+You can also [download the compiled and minified assets (CSS, JavaScript) from
+GitHub](docs/installation/installing-from-dist.md).
 
 #### Global Styles
```

Comment:
I'm not sure the global styles or javascript section make sense, since they used to be next to the scss import section. At the moment it's not clear what these relate to, or how you'd follow the guidance.